### PR TITLE
Add Additional Logging Output

### PR DESCRIPTION
**Pull request by bencap**
The Musher app has very little in terms of useful logging output, so this change
adds a few logging outputs at the info, debug, and error levels. These should
help future development and give us additional insights into the running application.

### CODE REVIEW

File: lib/husky_musher/app.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def set_cache_control(response):
 
 @app.errorhandler(404)
 def page_not_found(error):
+    app.logger.error(f'Page Not Found', exc_info=error)
```
Comment (kschwabefry) [inline lib/husky_musher/app.py:51]: I'm inclined to log this at an info level. Its useful to know about, e.g. could alert us to broken links. However, if a user is entering bogus URLs, in many cases that doesn't indicate an error in the eyes of the application.

**Review by kschwabefry — APPROVED** [review]
Left a suggestion for one of the new logs, but looks good overall! Thanks for adding some logging here.

File: lib/husky_musher/app.py (hunk 1 above)
Comment (bencap) [reply lib/husky_musher/app.py:51]: Makes a lot of sense, will update for Monday.

**Comment by bencap** [comment]
Deployed.

**Pull request merged**